### PR TITLE
npth: Add npth package

### PR DESCRIPTION
**Pull request by cheat-sc**
This commit adds npth package

### CODE REVIEW

File: recipes-debian/npth/npth_debian.bb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "New GNU Portable Threads library"
```
Comment (masami256) [inline recipes-debian/npth/npth_debian.bb:5]: @cheat-sc  could you add following fixed comment please.
```
# base recipe: 
# base branch: 
# base commit: 
```

Comment (cheat-sc) [reply recipes-debian/npth/npth_debian.bb]: I'm sorry I forgot to add them. I have added them.

Comment (cheat-sc) [reply recipes-debian/npth/npth_debian.bb:5]: I'm sorry I forgot to add them. I have added them.

Comment (masami256) [reply recipes-debian/npth/npth_debian.bb:5]: thanks

**Pull request merged**